Azure/sonic-buildimage
# [201911 SONIC-SWSS] update submodule

**- Why I did it**
Updating submodule for sonic-swss to get the changes to Azure 201911. The following were the commits that were part of this submodule.
[201911-SWSS]flushing FDB entries per VLAN when deleting VLAN (PR#https://github.com/Azure/sonic-swss/pull/1575)  9519fead3fc63972131de9cb8963a5aeacf7b23d

**- How I did it**
Merged sonic-swss changes for fix of per vlan fdb flush during vlan delete to Azure 201911. Now, updating the submodule changes.

**- How to verify it**
Configure a port part of 2 VLAN membership and delete it from 1 VLAN membership and delete the VLAN, the VLAN delete should be successful.

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**

The change is in SWSS Ports Orchagent code.

**- A picture of a cute animal (not mandatory but encouraged)**
